.circleci/config.yml

```diff
@@ -140,7 +128,22 @@ jobs:
       - run:
           name: run validation checks on all chains
           command: just validate ""
-      - notify-failures-on-validate-all
+      - notify-failures-on-main:
+          channel: C07GQQZDW1G # to slack channel `notify-superchain-validation-failures`
+  golang-promotion-test:
+    shell: /bin/bash -eo pipefail
+    executor:
+      name: go/default  # is based on cimg/go
+      tag: '1.21'
+    steps:
+      - checkout
+      - install-just
+      - install-foundry
+      - run:
+          name: run validation checks
+          command: just promotion-test
+      - notify-failures-on-main:
+          channel: C07GZVCCUS0 # to slack channel `notify-superchain-promotion-test-failures`
```

Comment:
Nit: could probably change the channel name to `notify-superchain-promotion-failures` (i.e. drop `test`) to reduce length and match naming convention of `notify-superchain-validation-failures`

Reply:
Good catch! Updated channel name and comment.